src/components/ProjectSettingsTabs/VariablesTab/VariableRow.tsx

```diff
@@ -66,3 +69,12 @@ const RowContainer = styled.div`
     max-width: 50%;
   }
 `;
+
+const OptionRow = styled.div`
+  display: flex;
+  margin-bottom: ${size.s};
+
+  > div {
+    flex-shrink: 0;
```

Comment:
[nit] It may be nice to add a bit of spacing between the two checkboxes
```suggestion
    flex-shrink: 0;
    margin-right: ${size.s};
```